src/pkg/utils/utils.go

```diff
@@ -0,0 +1,283 @@
+	if _, err := user.LookupGroup("sudo"); err == nil {
+		group = "sudo"
+	} else if _, err := user.LookupGroup("wheel"); err == nil {
+		group = "wheel"
+	}
+	return group
+}
+
+// GetMountPoint returns the mount point of a target.
+func GetMountPoint(target string) (string, error) {
+	cmd := exec.Command("df", "--output=target", target)
+	output, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+
+	options := strings.SplitAfter(string(output), "\n")
+
+	return strings.Trim(string(options[1]), "\n"), nil
+}
+
+// GetMountOptions returns the mount options of a target.
+func GetMountOptions(target string) (string, error) {
+	cmd := exec.Command("findmnt", "--noheadings", "--output", "OPTIONS", target)
+	output, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+
+	options := strings.SplitAfter(string(output), "\n")
+
+	return strings.Trim(options[0], "\n"), nil
+}
+
+// GetUID returns the user ID
+func GetUID() (string, error) {
+	currentUser, err := user.Current()
+	if err != nil {
+		return "", errors.New("Failed getting user information")
+	}
+	return currentUser.Uid, nil
+}
+
+// ShortID shortens provided id to first 12 characters.
+func ShortID(id string) string {
+	if len(id) > idTruncLength {
+		return id[:idTruncLength]
+	}
+	return id
+}
+
+// ReferenceCanBeID checks if the provided text matches a format for an ID.
+func ReferenceCanBeID(text string) bool {
+	matched, err := regexp.MatchString(`^[a-f0-9]\{6,64\}$`, text)
+	if err != nil {
+		logrus.Error(err)
+	}
+	return matched
+}
+
+// ReferenceHasDomain checks if the provided text has a domain definition in it.
+func ReferenceHasDomain(text string) bool {
+	i := strings.IndexRune(text, '/')
+	if i == -1 {
+		return false
+	}
+
+	// A domain should contain a top level domain name. An exception is 'localhost'
+	if !strings.ContainsAny(text[:i], ".:") && text[:i] != "localhost" {
+		return false
+	}
+
+	return true
+}
+
+// PathExists wraps around os.Stat providing a nice interface for checking an existence of a path.
+func PathExists(path string) bool {
+	if _, err := os.Stat(path); !os.IsNotExist(err) {
+		return true
+	}
+	return false
+}
+
+// UpdateContainerAndImageNames takes care of standardizing names of containers and images.
+//
+// If no image name is specified then the base image will reflect the platform of the host (even the version).
+// If no container name is specified then the name of the image will be used.
+//
+// If the host system is unknown then the base image will be 'fedora-toolbox' with a default version
+func UpdateContainerAndImageNames(containerName string, imageName string, release string) (string, string, string) {
+	imageDomain := ""
+	hostPlatform := GetHostPlatform()
+
+	if release == "" {
+		if hostPlatform == "fedora" {
+			release = GetHostVersionID()
+			if release == "rawhide" {
+				release = "32"
+			}
+		} else {
+			release = viper.GetString("RELEASE_DEFAULT")
+		}
+	}
+
+	if imageName == "" {
+		if hostPlatform == "fedora" {
```

Comment:
Isn't `imageName` the same for both *fedora* and *!fedora* ? Or am I misreading the code? :)

Reply:
I'm not sure if I follow here. Could you, please, elaborate?